### PR TITLE
[WIP] Enabling logging of oauth-server

### PR DESCRIPTION
**Pull request by EmilyM1**
Will be adding Deployment changes to make volume for oauth-server audit logs

### CODE REVIEW

File: bindata/oauth-openshift/deployment.yaml [hunk 1]
```diff
@@ -171,3 +174,5 @@ spec:
           configMap:
             name: v4-0-config-system-trusted-ca-bundle
             optional: true
+        - name: oauth-server-audit
```
Comment (stlaz) [inline bindata/oauth-openshift/deployment.yaml:177]: incomplete mount description?

File: bindata/oauth-openshift/deployment.yaml [hunk 2]
```diff
@@ -102,6 +102,9 @@ spec:
             - name: v4-0-config-system-trusted-ca-bundle
               readOnly: true
               mountPath: /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle
+            - name: oauth-server-audit
+              readOnly: true
```
Comment (stlaz) [inline bindata/oauth-openshift/deployment.yaml:106]: this does't make sense

File: bindata/oauth-openshift/deployment.yaml [hunk 3]
```diff
@@ -102,6 +102,9 @@ spec:
             - name: v4-0-config-system-trusted-ca-bundle
               readOnly: true
               mountPath: /var/config/system/configmaps/v4-0-config-system-trusted-ca-bundle
+            - name: oauth-server-audit
```
Comment (stlaz) [inline bindata/oauth-openshift/deployment.yaml:105]: see if https://github.com/openshift/cluster-authentication-operator/blob/master/bindata/oauth-apiserver/deploy.yaml#L36-L51 is needed here, too

**Review by stlaz — CHANGES_REQUESTED** [review]
/hold

Should be marked as WIP

File: bindata/oauth-openshift/deployment.yaml (hunk 1 above)
Comment (EmilyM1) [reply bindata/oauth-openshift/deployment.yaml:177]: Yeah, just added WIP, very early stages here

File: bindata/oauth-openshift/deployment.yaml [hunk 4]
```diff
@@ -49,6 +49,28 @@ spec:
...
+          terminationMessagePolicy: FallbackToLogsOnError
+          image: ${IMAGE}
+          imagePullPolicy: IfNotPresent
+          command: ['sh', '-c', 'chmod 0700 /var/log/oauth-server && touch /var/log/oauth-server/audit.log && chmod 0600 /var/log/oauth-server/*']
+          args:
+            - exec oauth-server osinserver \
+              --audit-log-path=/var/log/oauth-server/audit.log \
+              --audit-log-format=json \
+              --audit-log-maxsize=100 \
+              --audit-log-maxbackup=10 \
```
Comment (stlaz) [inline bindata/oauth-openshift/deployment.yaml:63]: please explain

**Comment by EmilyM1** [comment]
closed in favor of https://github.com/openshift/cluster-authentication-operator/pull/507/

**Pull request closed without merging**